Dockerfile

```diff
@@ -2,10 +2,10 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 
 # Application image
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
 
 COPY lib/AI-Agent.xml /opt/app/
-COPY build/libs/pip-subscription-management.jar /opt/app/
+COPY build/libs/$APP /opt/app/
 
 EXPOSE 4550
 CMD [ "pip-subscription-management.jar" ]
```

Comment:
Might be out of scope for this ticket - however the name of the JAR currently produced via gradle is **spring-boot-template.jar.**

I think that needs changing to match the above.